Main/BasicComponents/GradientBorder.qml

```diff
@@ -1,3 +1,23 @@
+/*
+    Copyright (C) 2019 Rob van den Berg <rghvdberg at gmail dot org>
```

Comment:
@rghvdberg probably needs to explicitly agree to the licensing of this file under LGPL

Reply:
Ah yes, thanks for catching that